.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
we use rome :)

Reply:
My editor kept making everything messy. Will remove